lib/elixir/lib/exception.ex

```diff
@@ -1997,11 +1997,11 @@ defmodule Protocol.UndefinedError do
 
   @impl true
   def message(%{protocol: protocol, value: value, description: description}) do
-    "protocol #{inspect(protocol)} not implemented for #{inspect(value)} of type " <>
+    "protocol #{inspect(protocol)} not implemented for #{inspect(value, pretty: true)} of type " <>
       value_type(value) <> maybe_description(description) <> maybe_available(protocol)
```

Comment:
I am wondering if we should change the message altogether to show the value at the end:

```suggestion
    "protocol #{inspect(protocol)} not implemented type " <> value_type(value) <>
      maybe_description(description) <> maybe_available(protocol) <>
      "\n\nGot value: #{inspect(value, pretty: true)}"
```

Reply:
I like it, that's preferable IMO